### DISCUSSION TITLE
Not able to pause the video after createVideoTexture

### DISCUSSION THREAD

**Discussion by Akhil-coditas** [google/model-viewer#4263] [Q&A]
so i have feature when on a press of button i need to pause the video that is being rendered on .glb 
also rewind and resume it.

![image](https://user-images.githubusercontent.com/106797543/235067519-3d4080ac-55a6-4c0e-8002-804c9c6bb979.png)

this is how im rendering the video texture

and regarding pausing the video 
```
  var modelViewer = document.querySelector("#model-viewer");
     modelViewer.pause();
```

also used 

```
  const sceneSym = Object.getOwnPropertySymbols(modelViewer).find(x => x.description === "scene");
  const scene = modelViewer[sceneSym]
  console.log(scene);
```

but can i pause video using scene ? if yes can you provide me a snippet if possible

but it doesn't work. 
is there a way i can get access to three js being used in model viewer then pause it ? or  any other recommended way?

**Comment by elalish** [comment]
For play/pause/seek on the video itself, you need the HTML video element, which you access as shown here: https://modelviewer.dev/docs/#entrydocs-scenegraph-methods-createVideoTexture Use e.g. `videoTexture.source.element.pause()` with the web API: https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement
